src/zenml/zen_server/auth.py

```diff
@@ -94,7 +94,7 @@ def authenticate_credentials(
     if user_name_or_id:
         try:
             user = zen_store().get_auth_user(user_name_or_id)
-            user_model = zen_store().get_user(user_name_or_id=user_name_or_id)
+            user_model = zen_store().get_myself(user_name_or_id=user_name_or_id)
```

Comment:
why would we still need to specify `user_name_or_id` if it's get_myself shouldn't the function be getting the current active user by default?

Reply:
Ah so the difference between `get_myself() and `get_user()` is not that `get_myself()` returns a fixed user but rather that the returned model has more fields than the default user response.